src/Main.java

```diff
@@ -2,6 +2,41 @@
 // click the <icon src="AllIcons.Actions.Execute"/> icon in the gutter.
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello and welcome!");
+        //Task1
+        System.out.println("Задача 1");
+        int[] salary = {3000, 1000, 5000, 9000, 7650};
+        int sum = 0;
+        for (int s : salary) {
+            sum = sum + s;
+        }
+        System.out.println("Сумма трат за месяц составила " + sum + " рублей");
+        //Task2
+        System.out.println("Задача 2");
+        int[] spend = {100, 211, 303, 505, 8};
+        int maxSpend = spend[1];
```

Comment:
Обычно принято к нулевому приравнивать
```suggestion
        int maxSpend = spend[0];
```